llama_stack/distribution/build_container.py

```diff
@@ -0,0 +1,373 @@
+#!/usr/bin/env python3
```

Comment:
What's the rationale for making this a standalone script and not a module we can directly call?